.storybook/main.js

```diff
@@ -48,80 +48,105 @@ const defaultRazzleOptions = {
   staticCssInDev: false,
   emitOnErrors: false,
   disableWebpackbar: false,
-  browserslist: ['>1%', 'last 4 versions', 'Firefox ESR', 'not ie 11', 'not dead']
+  browserslist: [
```

Comment:
Although the change seems legit, it would go to it's own PR. It seems that this file never was formatted properly, and the CI is not catching it.